samples/boards/nordic/clock_control/configs/audiopll.conf

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_CLOCK_CONTROL=y
```

Comment:
also `CONFIG_SAMPLE_CLOCK_FREQUENCY_HZ` should be defined here.

Reply:
The frequency is not settable, so the frequency has no impact here, the spec is entirely ignored :)